recipes/h3pandas/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The `LICENSE` is already included in the PyPI tarball, so it shouldn't be manually included here.

Reply:
Thanks for the review, I removed the `LICENSE` file.